### DISCUSSION TITLE
Qfield recording in *.m4a but QGIS not able to play *.m4a

### DISCUSSION THREAD

**Discussion by JP-vie** [opengisch/QField#5787] [Q&A]
When recording audio data with Qfield it is stored on my device as *.m4a files. When syncing the project back to Qgis they can not be played. The files appear as not vaild in the attribute form. I need to convert them to *.mp3 so they are readable for QGIS, is there a way to define in which file-format Qfield records audio files, or implement a *.m4a codec into Qgis?

thanks for any help
J.

**Comment by beanzmo** [comment]
Hi @JP-vie 
Have you configured your attribute field according to the https://docs.qfield.org/de/how-to/attributes-form/?h=audio#widget-fur-anhange-konfigurieren
What version of QGIS/QField are you using?
B

**Reply by JP-vie** [reply]
Hi @beanzmo,
Yes I configured the attribute field accordingly. 
QGIS version 3.34, Qfield 3.4.4

J

**Comment by beanzmo** [comment]
Hi @JP-vie  - try the obvious first and then look for other solutions. I asked internally and it was confirmed. Currently it cannot be done but this is related to the way that QGIS itself is currently written and cannot handle all the formatting changes when jumping from different operating systems like linux  (for example). This may change in the fututreif QT 6 comes in but for now there is no immediate solution. 

One could think about - in QField, however to allow users to choose between different formats. This would come with professional costs though. Estimated work time is 2 days.
